components/eamxx/src/control/atmosphere_surface_coupling_exporter.hpp

```diff
@@ -21,6 +21,14 @@ namespace scream
  * in its list of subcomponents (the AD should make sure of this).
 */
 
+// enum to track how exported fields will be set.
+enum ExportType {
+  EAMXX      =  0,  // Variable will be derived from EAMxx state
+  PRESCRIBED =  1,  // Variable will be set given data from a file
+  CONSTANT   =  2   // Set variable to a constant value
```

Comment:
Would `FILE = 1` make more sense because several places we have `prescribed_constants`. I'm guessing when this option becomes available, the parameter list will have `prescribed_constant` and `prescribed_from_file` or something like that, so "prescribed" applies to both.

Reply:
Good point, I change to `FROM_FILE`, I wanted to avoid `FILE` alone since I think that file is a key word for other things too.